setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+    git_requirements = [
+        f"{req.split('#egg=')[1]} @ {req}"
+        for req in requirements if req.startswith('git')
+    ]
+    normal_requirements = [req for req in requirements if not req.startswith('git')]
+
+setup(
+    name='wam',
+    version='1.0',
```

Comment:
Why didn't you use the current version here? (v0.1.2)

Reply:
Rapidly typed it. This should be adapted...

Reply:
Done in 972faddffa4fe2356f4badace8d42602a090345a. Also added some more meta infos